dev_guide/service_accounts.adoc

```diff
@@ -213,6 +213,35 @@ Tokens:            	robot-token-8bhpp
 ----
 ====
 
+== Checking if Pod Creation is Possible
+
+Using the `scc-review` and `scc-subject-review` options, you can check if a user, service account, or group can create a pod. The command outputs the security context constraints that admit the resource.
+
+To check if a specific service account can create a pod in a specific file:
+
+----
+$ oc policy scc-review -z <service-account1>,<service_account2> -f myfile.yaml
+----
+
+To check if a user with the system:serviceaccount:username:default service account can a create a pod:
+
+----
+$  oc policy scc-review -z system:serviceaccount:username:default -f my_resource.yaml
+----
+
+To check whether a user can create a pod in a specific file:
+
+----
+$ oc policy scc-subject-review -u <username> -f myfile.yaml
+----
+
+To check if a user belonging to a specific group can create a pod in a specific file:
+
+----
+$ oc policy scc-subject-review -u <username> -g <groupname> -f myfile.yaml
+----
+
+
```

Comment:
These commands are useful for debugging SCC problems and I'm suggesting to move this chapter to SCC documentation where user will have more chances to find it: https://docs.openshift.org/latest/admin_guide/manage_scc.html